### ISSUE TITLE
NumericTextBox should disable autocomplete attribute of the input

### ISSUE THREAD

**Issue by danielkaradachki** [telerik/kendo-angular#771]

**Comment by dgroh** [comment]
Which release is this fix planned for?

**Comment by danielkaradachki** [comment]
The next version should contain the fix.

**Comment by siyka-marinova** [comment]
The fix is currently available in the latest dev. version: `@progress/kendo-angular-inputs@v1.0.8-dev.201707250902`.
To get this version, specify in package.json:
`"@progress/kendo-angular-inputs": "dev"`

**Comment by dgroh** [comment]
The problem is still there for Edge.

**Comment by dgroh** [comment]
The package was not updated to 1.0.8, see image:

![image](https://user-images.githubusercontent.com/16785357/28912978-b08776d2-7836-11e7-8521-bed2d360fc74.png)

**Comment by ggkrustev** [comment]
@dgroh, Kendo packages were moved to NPMJS. I would suggest you check the .npmrc file. More details here:

https://github.com/telerik/kendo-angular/issues/712

Running `npm view @progress/kendo-angular-inputs` show that the version in question is there.

![package_version](https://user-images.githubusercontent.com/495188/28917247-a0c79ca6-784d-11e7-9f95-0686affb02cd.png)

**Comment by dgroh** [comment]
Hi @ggkrustev,

`.npmrc` is already empty

![image](https://user-images.githubusercontent.com/16785357/28920774-879be092-7853-11e7-9ff0-ae8e953e55fd.png)

**Comment by ggkrustev** [comment]
Great! In that case, could you try to install a specific version then?
```
npm install @progress/kendo-angular-inputs@1.0.8-dev.201707250902

//or 

npm install @progress/kendo-angular-inputs@1.1.0-dev.201707271519
```

Let us know if the issue still persists after the latest dev version is installed.

**Comment by dgroh** [comment]
It still persists:

![image](https://user-images.githubusercontent.com/16785357/28922462-84de1800-785a-11e7-9475-d28a639f6376.png)

This is my package json

```json
{
  "name": "app",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^4.1.3",
    "@angular/common": "^4.0.0",
    "@angular/compiler": "^4.0.0",
    "@angular/core": "^4.0.0",
    "@angular/forms": "^4.0.0",
    "@angular/http": "^4.0.0",
    "@angular/platform-browser": "^4.0.0",
    "@angular/platform-browser-dynamic": "^4.0.0",
    "@angular/router": "^4.0.0",
    "@progress/kendo-angular-dropdowns": "^1.0.10",
    "@progress/kendo-angular-inputs": "^1.0.6",
    "@progress/kendo-angular-intl": "^1.2.1",
    "@progress/kendo-angular-l10n": "^1.0.0",
    "@progress/kendo-angular-layout": "^1.0.3",
    "@progress/kendo-theme-default": "^2.37.1",
    "core-js": "^2.4.1",
    "ng2-translate": "^5.0.0",
    "rxjs": "^5.1.0",
    "zone.js": "^0.8.4"
  },
  "devDependencies": {
    "@angular/cli": "1.0.3",
    "@angular/compiler-cli": "^4.0.0",
    "@types/jasmine": "2.5.38",
    "@types/node": "~6.0.60",
    "codelyzer": "~2.0.0",
    "jasmine-core": "~2.5.2",
    "jasmine-spec-reporter": "~3.2.0",
    "karma": "~1.4.1",
    "karma-chrome-launcher": "~2.1.1",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "^0.2.0",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "karma-junit-reporter": "^1.2.0",
    "karma-phantomjs-launcher": "^1.0.4",
    "node-sass": "^4.5.3",
    "phantomjs-prebuilt": "^2.1.14",
    "protractor": "~5.1.0",
    "ts-node": "~2.0.0",
    "tslint": "~4.5.0",
    "typescript": "~2.2.0"
  }
}
```

**Comment by ggkrustev** [comment]
The error in the terminal says the dev version (1.0.8) is invalid. I am pretty sure that NPM tries to install it from the private NPM registry. Could you try the suggestions in the linked issue and particularly this one:

https://github.com/telerik/kendo-angular/issues/712#issuecomment-314124396

**Comment by tsvetomir** [comment]
Fixed in `@progress/kendo-angular-inputs@1.1.0`
